contracts/index_executor/src/lib.rs

```diff
@@ -727,12 +724,13 @@ mod index_executor {
                 .unwrap();
 
             // Deploy Executor
-            ExecutorRef::default()
-                .code_hash(hash)
-                .endowment(0)
-                .salt_bytes([0u8; 0])
-                .instantiate()
-                .expect("failed to deploy Executor")
+            // ExecutorRef::default()
+            //     .code_hash(hash)
+            //     .endowment(0)
+            //     .salt_bytes([0u8; 0])
+            //     .instantiate()
+            //     .expect("failed to deploy Executor")
+            Executor::default()
```

Comment:
old code should just be deleted, not commented

Reply:
Done